sources/Interop/Windows/um/wingdi/Windows.cs

```diff
@@ -1335,7 +1335,7 @@ public static unsafe partial class Windows
 
         [DllImport("opengl32", ExactSpelling = true, SetLastError = true)]
         [return: NativeTypeName("PROC")]
-        public static extern delegate* unmanaged<int> wglGetProcAddress([NativeTypeName("LPCSTR")] sbyte* param0);
+        public static extern IntPtr wglGetProcAddress([NativeTypeName("LPCSTR")] sbyte* param0);
```

Comment:
is this loss of the delegate type intentional?

Reply:
Yes. `PROC` (and `FARPROC`) is meant to be an opaque function pointer. The actual type isn't known and you need to cast it to the correct signature.